src/main/java/software/amazon/awssdk/crt/mqtt5/packets/SubscribePacket.java

```diff
@@ -307,6 +307,40 @@ public SubscribePacketBuilder withUserProperties(List<UserProperty> userProperti
          */
         public SubscribePacketBuilder() {}
 
+        /**
+         * Creates a new SubscribePacketBuilder with one subscription defined.
+         *
+         * @param subscription The subscription to add within the SubscribePacket.
+         */
+        public SubscribePacketBuilder(Subscription subscription) {
+            withSubscription(subscription);
+        }
+
+        /**
+         * Creates a new SubscribePacketBuilder with one subscription defined.
+         *
+         * @param topicFilter The topic filter to subscribe to
+         * @param qos The maximum QoS on which the subscriber will accept publish messages
+         * @param noLocal Whether the server will not send publishes to a client when that client was the one who
+         * sent the publish
+         * @param retainAsPublished Whether messages sent due to this subscription keep the retain flag preserved
+         * on the message
+         * @param retainHandlingType Whether retained messages on matching topics be sent in reaction to this subscription
+         */
+        public SubscribePacketBuilder(String topicFilter, QOS qos, Boolean noLocal, Boolean retainAsPublished, RetainHandlingType retainHandlingType) {
```

Comment:
Question the need for this one tbh

Reply:
I had some doubts about adding this constructor, I'll remove it